### ISSUE TITLE
[PATCH tbb] tbb: Autospec creation for update from version 2019.u9 to  version 2021.9.0

### ISSUE THREAD

**Issue by schiele** [clearlinux/distribution#2908]
tbb shipped with ClearLinux is quite outdated. I have some software that expects to see the newer oneAPI interface style, as such an update might help moving forward.

This is what I produced so far: [for-review.txt](https://github.com/clearlinux/distribution/files/11710168/for-review.txt)

**Comment by fenrus75** [comment]
The sad problem is that TBB broke ABI/API and did it in  a bad way (old and
new don't coexist)... so we have no nice path forward; last time I tried
this upgrade half the OS brokes since not all users could cope with the new
TBB

**Comment by schiele** [comment]
Oh, sad to hear. Are you talking about software packaged in ClearLinux that broke or did you get feedback from users with external software that got problems? If our concern is mostly about packages included with ClearLinux, that would mean if we just found a solution to all of those packages, we could proceed with the update then? I just checked and it seems we have less than 10 packages referring to tbb in their spec files, such that I might consider having a look into them and check whether there is an easy solution to adapt them.
BTW: Since I didn't find a clear answer to this yet in the ClearLinux documentation: Is there kind of a public CI system to test the impact of package changes on the overall distribution for ClearLinux, similar to what openSUSE offers with their [Build Service](https://build.opensuse.org/)?
